### ISSUE TITLE
sign off not always correct in notifications emails

### ISSUE THREAD

**Issue by JChipault** [USGS-WiM/whispers#906]
It seems the sign off that is being appended to the dynamic message in the notifications emails is sometimes this:

Thank you very much for your interest in the Wildlife Health Information Sharing Partnership (WHISPers). Currently, users from state, federal, and tribal natural resources agencies with management authority over species and/or land are being enrolled as WHISPers Partners.

Based on the information provided, you have been assigned the role of Public User. You have access to all publicly-available historical and current WHISPers event information and can now log in with your username and password and save your searches of event data to your user dashboard.

Wildlife professionals involved in collaborative projects with state, federal, or tribal natural resource management agencies can be sponsored for enrollment as a WHISPers "Affiliate," which allows the WHISPers partner to grant access to detailed information for specific events. If you'd like to be an Affiliate, please talk with your collaborating partner then email whispers@usgs.gov with the name and agency of the state, federal, or tribal sponsor(s) you have been in contact with regarding being designated an Affiliate user. See the USGS Wildlife Health Center website for more information about roles, as well as to find user guides and metadata (https://www.usgs.gov/nwhc/whispers).

Thanks again!

It should be: 
WHISPers Administration Team 
USGS National Wildlife Health Center
6006 Schroeder Rd.
Madison, WI  53711
website:  whispers.usgs.gov
email:  whispers@usgs.gov

with no hard enters between lines and the website as a link. (sometimes when this sign off is used, there's a lot of space between lines and I don't think I've seen one that has a live link to the website)

**Comment by JChipault** [comment]
this one feels too spaced out, it takes up a lot of room
![image](https://user-images.githubusercontent.com/49287076/75592378-c60add00-5a47-11ea-878a-18ec9f9a7cc8.png)

**Comment by JChipault** [comment]
here's a better screen shot. the main message here is nicely spaced but then the sign off is too spaced out between lines.
![image](https://user-images.githubusercontent.com/49287076/75592424-e8045f80-5a47-11ea-827d-cae8c593eebd.png)

**Comment by aaronstephenson** [comment]
Yes, thanks, that signoff you saw in the emails from yesterday and earlier today saying "Thank you for your interest" etc was a mistake I made in the boilerplate value of the configuration table. This was corrected earlier this afternoon to be the proper sign off.

As for the spacing, feel free to make whatever edits you like. This particular value is stored in the whispers_configuration table, it is the record with name "email_boilerplate".
